libs/game/game.ts

```diff
@@ -58,13 +58,9 @@ namespace game {
 
     function showDialogBackground(h: number, c: number) {
         const top = (screen.height - h) >> 1;
-        if (screen.isMono) {
-            screen.fillRect(0, top, screen.width, h, 0)
-            screen.drawLine(0, top, screen.width, top, 1)
-            screen.drawLine(0, top + h - 1, screen.width, top + h - 1, 1)
-        } else {
-            screen.fillRect(0, top, screen.width, h, c)
```

Comment:
no more color?

Reply:
It's only the background, I think black with the white lines look better. It was all black&white for a while because of `screen.isMono` bug. @riknoll is going to re-do these anyways

Reply:
yup